code/modules/surgery/tools.dm

```diff
@@ -97,4 +97,57 @@
 
 /obj/item/weapon/surgical_drapes/attack(mob/living/M, mob/user)
 	if(!attempt_initiate_surgery(src, M, user))
-		..()
\ No newline at end of file
+		..()
+		
+/obj/item/weapon/organ_storage //allows medical cyborgs to manipulate organs without hands
+	name = "organ storage bag"
+	desc = "A container for holding body parts."
+	icon = 'icons/obj/storage.dmi'
+	icon_state = "evidenceobj"
+
+/obj/item/weapon/organ_storage/afterattack(obj/item/I, mob/user, proximity)
+	if(!proximity)
+		return
+	if(contents.len)
+		to_chat(user, "<span class='notice'>[src] already has something inside it.</span>")
+		return
+	if(isorgan(I))
+		var/obj/item/organ/O = I
+		if(O.status != ORGAN_ORGANIC)

```

Comment:
Trusting a borg to do your surgery is indeed less dangerous as with a human. There is risk of a rogue borg killing you during the surgery unless you built it yourself, unlinked, and never let it out of your sight. You could mitigate the risk of being murdered by your surgeon with borgs, as you have a way of preventing them ever being antag.

I understand your stance here, since cybernetic implants grant you nice buffs, but at the risk of trusting a human to not murder you, and know what he is doing. The hardest part is getting some lazy sod to even help you! A mediborg is much more likely to help you with surgery because of the player's servant mentality (and requirement by laws).

Still, this change even as it is shall give mediborgs a nice bump in their usefullness.